### DISCUSSION TITLE
密码重置 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#992] [Announcements]
# 密码重置 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/ghippo/user-guide/password/

**Comment by liyuerich** [comment]
没有安全策略的介绍

**Reply by yitingdc** [reply]
用 https://gitlab.daocloud.cn/ndx/ghippo/-/issues/635 跟踪

**Comment by windsonsea** [comment]
参阅：https://docs.daocloud.io/ghippo/user-guide/platform-setting/security/
